symbulate/distributions.py

```diff
@@ -618,16 +657,24 @@ def __init__(self,
                  sd1=1.0, sd2=1.0, corr=0.0,
                  var1=None, var2=None, cov=None):
 
-        if corr is not None and not (-1 <= corr < 1):
+        if not (-1 <= corr <= 1):
             raise Exception("Correlation must be "
                             "between -1 and 1.")
 
         self.mean = [mean1, mean2]
 
+        if (sd1 < 0):
+            raise Exception("sd1 cannot be less than 0")
+        if (sd2 < 0):
+            raise Exception("sd2 cannot be less than 0")
+
+
         if var1 is None:
             var1 = sd1 ** 2
         if var2 is None:
             var2 = sd2 ** 2
+        if (var1 < 0) or (var2 < 0):
+            raise Exception("var1 or var2 cannot be negative")
         if cov is None:
             if var1 * var2 < 0:
```

Comment:
I don't get it. This check seems unnecessary to me. You've already verified above that var1 and var2 are both greater than 0, so of course var1 * var2 are also > 0.